src/fruity/fruity-host-session.vala

```diff
@@ -812,6 +809,12 @@ namespace Frida {
 
 			return yield attach_via_gadget (pid, gadget_details, cancellable);
 		}
+		
+		private async LLDB.Client start_lldb_service (Fruity.LockdownClient lockdown, Cancellable? cancellable)
+				throws Error, IOError, Fruity.LockdownError, LLDB.Error {

```

Comment:
```suggestion
				throws Fruity.LockdownError, LLDB.Error, IOError {
```

- Assuming `Error` is not thrown here.
- Moving IOError last as it's only for when `cancellable` causes `IOError.CANCELLED`.